LICENSE

```diff
@@ -0,0 +1,29 @@
+                                New BSD License
+
+
+
+Copyright © 2007-2014, Ivan Enderlin.
```

Comment:
Not an up-to-date copyright. My fault. http://hoa-project.net/LICENSE is now up-to-date, see https://github.com/hoaproject/W3/pull/72.